collect_app/src/main/java/org/odk/collect/android/preferences/PreferencesFragment.java

```diff
@@ -129,6 +133,39 @@ public boolean onPreferenceChange(Preference preference, Object newValue) {
         }
     }
 
+    private void initLanguagePrefs() {
+        final ListPreference pref = (ListPreference) findPreference(KEY_LANGUAGE);
+        final LocaleHelper localeHelper = new LocaleHelper();
+        TreeMap<String, String> languageList = localeHelper.getEntryListValues();
+        pref.setEntryValues(languageList.values().toArray(new String[0]));
+        pref.setEntries(languageList.keySet().toArray(new String[0]));
+
+        if (pref != null) {
```

Comment:
This check is too late. `pref` can never be `null` here because if it were, the previous two lines of code would have crashed!